device.go

```diff
@@ -232,7 +232,16 @@ func (device *BlockDevice) attach(h hypervisor, c *Container) (err error) {
 
 		device.DeviceInfo.Hotplugged = false
 	} else {
-		if err = h.hotplugAddDevice(drive, blockDev); err != nil {
+		var err error
+		for _, b := range c.pod.config.VMConfig.Bridges {
+			err = b.hotplugAddPCIDevice(h, drive, blockDev)
```

Comment:
That's exactly what I am pointing in a previous commit as not the right way to proceed. hotplugAddPCIDevice() should be called from the hypervisor implementation (no need for `h` as parameter in that case), and we should not modify the call here. Hypervisor implementation should be able to detect if it needs to hotplug a PCI device or not I think.

Reply:
we need to save PCI bridges in the pod state because before trying to hot plug any device we must know that slots are available, is the hypervisor structure saved in the pod state?

Reply:
@devimc currently, we don't save any data related to hypervisor in a state file (relates to run-time which is different from config files storing the Hypervisor config). But I think that if you need to store some informations so that you can reuse them across different containers, you should maybe create a new state file for the hypervisor.
Before you do that, I'd like to make sure I understand properly what you need.

Reply:
@sboeuf do we store the POD state somewhere? also where do we store details about the devices attached to the PODs and containers.